### PR TITLE
Analytics

### PR DESCRIPTION
**Pull request by ShahAnuj2610**
UI:
image

### CODE REVIEW

File: components/SearchSandbox/index.js [hunk 1]
```diff
@@ -283,7 +288,9 @@ class SearchSandbox extends Component {
 		const {
 			appId, appName, url, credentials, attribution, customProps,
 		} = this.props; // prettier-ignore
-		const { componentProps, mappings, version } = this.state;
+		const {
+ componentProps, mappings, version, analytics,
```
Comment (jyash97) [inline components/SearchSandbox/index.js]: Formatting  is not right here

File: components/SearchSandbox/index.js [hunk 2]
```diff
@@ -155,6 +156,10 @@ class SearchSandbox extends Component {
 		return !!searchState;
 	}
 
+	toggleAnalytics = () => this.setState(prevState => ({
+			analytics: !prevState.analytics,
+		}));
```
Comment (jyash97) [inline components/SearchSandbox/index.js]: Formatting issue

File: components/SearchSandbox/index.js [hunk 3]
```diff
@@ -283,7 +290,8 @@ class SearchSandbox extends Component {
 		const {
 			appId, appName, url, credentials, attribution, customProps,
 		} = this.props; // prettier-ignore
-		const { componentProps, mappings, version } = this.state;
+		// eslint-disable-next-line object-curly-newline
```
Comment (jyash97) [inline components/SearchSandbox/index.js]: Why we have disabled these rules? Was it because prettier was messing up formatting?

Comment (ShahAnuj2610) [reply components/SearchSandbox/index.js]: Yes, it was giving eslint warning, solving that warning results in bad formatting

Comment (jyash97) [reply components/SearchSandbox/index.js]: Do `//prettier-ignore` instead

**Comment by jyash97** [comment]
@siddharthlatest Do lmk if the UI looks good I can merge this in.

**Pull request merged**